interface/themes/login.css

```diff
@@ -221,3 +221,8 @@ button, input.button, ul.button-bar li a,  x:-moz-any-link { padding: .35em 1em;
        width:296;
        height:72;
 }
+
+/* :..[ Registration Popup ]..: */
+.ui-widget > .ui-dialog-buttonpane button {
+  background: #2672ec;
+}
```

Comment:
I may migrate this to the main themes script and then may not need to include the login.css script in the about screen:
(I can do this minor change when bring the code in)